source/ref_trainings.rst

```diff
@@ -32,19 +32,19 @@ General training structure
         #   Name - spec.model.name
         #   Version - spec.model.version
         #   RandomUUID - a random UUID v4, for example be17d12d-df43-4588-99e7-56a0db3cad77
-        artifactNameTemplate: {{ .Name }}-{{ .Version }}-{{ .RandomUUID }}.zip
+        artifactNameTemplate: {{ .Name }}-{{ .Version }}-{{ .RandomUUID }}.zip  # optional
```

Comment:
Put the comment above the line.

Reply:
OK, then I delete this comment because this already mentioned above. I just decided to explicitly indicate what exactly optional.
![image](https://user-images.githubusercontent.com/49116723/74322237-4d8bf700-4d8c-11ea-9f70-c00ef5870bc2.png)